src/main/java/pl/wedding/management/model/employee/EmployeeDto.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.model.employee;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Builder
+@Getter
+class EmployeeDto {
+    private long id;
+    private String name;
+    private String surname;
+    private String workPlace;
```

Comment:
workPlace? to znaczy co? ul. Sienkiewicza? Poznań, ul.Sienkiewicza? a może "Restauracja Czerwony Kapturek"?